kuwala/core/canvas/src/pages/DataSourcePreview.js

```diff
@@ -0,0 +1,194 @@
+import React from "react";
+import Header from "../components/Header";
+import {useLocation, Link} from "react-router-dom";
+import {useStoreState} from "easy-peasy";
+
+import ListSVG from "../icons/list.svg"
+import ArrowRight from "../icons/arrow-right-solid.svg"
+import ArrowDown from "../icons/arrow-down-solid.svg"
+
+export default () => {
+
+    const location = useLocation()
+    const dataIndex = location.state.index
+    const { dataSource } = useStoreState((state) => state.canvas);
+    const selectedSource = dataSource[dataIndex]
+
+    const renderDataSourceConfig = () => {
+        if(!selectedSource) {
+            return (
+                <div>
+                    Undefined data source, something is wrong.
+                </div>
+            )
+        }else {
+            return (
+                <div className={'flex flex-row bg-white border-2 border-kuwala-green rounded-lg h-full w-full'}>
+                    <div className={'flex flex-col bg-white w-1/6 border-2 border-kuwala-green'}>
+                        <div className={'bg-kuwala-green w-full px-4 py-2 text-white font-semibold'}>
+                            Database: Kuwala
```

Comment:
This should be set dynamically based on the data source configuration

Reply:
Yes, this feature is still WIP and only a layout.